src/features/GuidedTour/GuidedTour.tsx

```diff
@@ -100,7 +100,7 @@ export function GuidedTour({
         },
       },
       {
-        target: "#root div[role=main]",
+        target: "#root > div",
```

Comment:
This does not work for Storybook 7 and 8. The elements structure changed and there's no `role=main` anymore. This needs to be tweaked and tested in both Storybook 7 and 8 (via a canary release)